ci/build.yml

```diff
@@ -73,9 +73,10 @@ suite-web build stable codesign:
 suite-web-landing build dev:
   stage: build
   script:
+    - apt install chromium
```

Comment:
We shouldn't do it like this, if you need chromium to be a part of the pipeline for this, let's add it to our base image so we're not pulling it all the time.